core/src/main/java/com/scalar/db/api/DistributedTransaction.java

```diff
@@ -69,8 +69,8 @@ public interface DistributedTransaction extends TransactionCrudOperable {
   /**
    * Commits a transaction.
    *
-   * @throws CommitConflictException if conflicts happened. You can retry the transaction in this
-   *     case
+   * @throws CommitConflictException if a transaction conflict occurs, You can retry the transaction
```

Comment:
```suggestion
   * @throws CommitConflictException if a transaction conflict occurs. You can retry the transaction
```

Reply:
Ah it's not typo actually. The format `@throws XXXException if ...` is frequently used in this project.